core/flow/tc_throttling.go

```diff
@@ -39,26 +36,30 @@ func (c *ThrottlingChecker) DoCheck(_ base.StatNode, batchCount uint32, threshol
 	}
 	// Here we use nanosecond so that we could control the queueing time more accurately.
 	curNano := util.CurrentTimeNano()
+
+	statIntervalNs := uint64(c.BoundOwner().BoundRule().StatIntervalInMs) * util.UnixTimeUnitOffset
```

Comment:
"statIntervalNs" could be field of ThrottlingChecker and fill "statIntervalNs" when new  ThrottlingChecker

Reply:
Addressed.